LuckyDragon/app/src/main/java/com/example/luckydragon/Models/User.java

```diff
@@ -129,6 +127,23 @@ public void save() {
                 });
     }
 
+    public void deleteUser(String deviceId) {
+        db.collection("users")
+                .document(deviceId)
+                .delete();
+    }
+
+    public void deleteOrganizerEvents(String deviceId) {
```

Comment:
make part of deleteUser if the user is an organizer to minimize the public methods

might want to make it delete the event from the eventlist aswell

Reply:
Done